cactusbot/handlers/command.py

```diff
@@ -23,27 +23,56 @@ def __init__(self, channel):
         super().__init__()
 
         self.channel = channel
-        self.api = CactusAPI(channel)
+        self.api = CactusAPI(channel.lower())  # FIXME
```

Comment:
@RPiAwesomeness It's important for a stable API release that `/user/2Cubed` works, in addition to `/user/2cubed`. (Both should access the same resources - having a user with name `2Cubed` *and* a separate one with name `2cubed` should not be possible.) However, that can be dealt with after the feature freeze.